### ISSUE TITLE
Error with automatic renew SSL certificate

### ISSUE THREAD

**Issue by MarcoAppleby** [globaleaks/GlobaLeaks#3248]
Hi,

I have received this email (in Italian language there is a misprint).:
```
Il rinnovo automatico del certificato HTTPS previsto per oggi è appena fallito.

Il sistema effetturerà automaticamente nuovi tentativi.
```

It is possibile to communicate more information about the error because I don't know where is the problem with the automatic renew of SSL certificate.

With the manual renew, no error occurred.

Thanks

**Comment by evilaliv3** [comment]
Dear @MarcoAppleby 

Thank you for reporting this. 
We are aware that time to time renewals are failing and we are still trying to understand the reasons.

**Comment by evilaliv3** [comment]
@MarcoAppleby : in the reality i've found no issues in the current release.

Which version of the software are you using? would you please upload a copy of the file /var/globaleaks/log/globaleaks.log and access.log eventually removing data that you consider sensible?

**Comment by MarcoAppleby** [comment]
Hi Giovanni,

I have installed the latest release.

The first error was found with the 4.9.5 version but it is been founded again with the 4.9.9.

Now, I have renewed the ssl manually so I haven't a new error mail to share you.

In attachment i send you the logs.

[access.log](https://github.com/globaleaks/GlobaLeaks/files/9130192/access.log)
[globaleaks.log](https://github.com/globaleaks/GlobaLeaks/files/9130193/globaleaks.log)

**Comment by evilaliv3** [comment]
@MarcoAppleby : This issue is solved in the current release 4.10.14.

The issue was due to the fact that letsencrypt is overloaded at 00:00 UTC.
We modified our renewal schedule to avoid this specific time and apply a random delay.
